src/client/src/bootstrapper.js

```diff
@@ -47,7 +47,7 @@ class Bootstrapper {
   startServices() {
     let user:User = FakeUserRepository.currentUser;
     let serverEndPointUrl = config.overwriteServerEndpoint ? config.serverEndPointUrl : location.hostname;
-    let url = 'ws://' + serverEndPointUrl + ':8080/ws', realm = 'com.weareadaptive.reactivetrader';
+    let url = 'ws' + (location.protocol === 'https:' ? 's:' : ':') + '//' + serverEndPointUrl + ':8080/ws', realm = 'com.weareadaptive.reactivetrader';
```

Comment:
This line is getting a bit insane now, can we break it into a method `_createAutobahnConnectionProxy()` and within there construct the url string using a string template, i.e. let url = `${protocol}://${serverEndPointUrl}:8080/ws'.

Makes it a bit more readable and removes clutter.